index.js

```diff
@@ -0,0 +1,50 @@
+function validatePassword(valid) {
+  if (passwordLength(valid) === true && passwordLowerCase(valid) === true && passwordUpperCase(valid) === true && passwordNumericValue(valid) === true && passwordSpecialCharacter(valid) === true) {
+    return true
+  }
+  else {
+    return false
+  }
+}
+function passwordLength(valid) {
+  if (valid.length >= 8) {
+    return true
+  }
+  else {
+    return false
+  }
```

Comment:
You don't need an `if` when returning true/false you can just return the condition which is already true/false.

```JavaScript
function isValidLength(password) {
  return password.length >= 8
}
```